### PR TITLE
launchd: Handle deprecated APIs in plistlib

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

* readPlist and writePlist are deprecated from plistlib.
  Handle this in the module code.

Fixes: #1552

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/1552_launchd.yml
plugins/modules/system/launchd.py

### CODE REVIEW

File: changelogs/fragments/1552_launchd.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- launchd - handle deprecated APIs like readPlist and writePlist in plistlib (https://github.com/ansible-collections/community.general/issues/1552).
```
Comment (felixfontein) [inline changelogs/fragments/1552_launchd.yml:2]:
```suggestion
- launchd - handle deprecated APIs like ``readPlist`` and ``writePlist`` in ``plistlib`` (https://github.com/ansible-collections/community.general/issues/1552).
```

**Review by felixfontein — COMMENTED** [review]
Change itself looks good.

**Comment by felixfontein** [comment]
recheck

**Review by felixfontein — APPROVED** [review]

**Comment by Akasurde** [comment]
recheck

**Comment by felixfontein** [comment]
@Akasurde thanks for fixing this!

**Comment by martinm82** [comment]
@Akasurde thanks a lot for fixing this. I had unfortunately no time to look at the PR.

**Pull request merged**